src/simd/sse.cpp

```diff
@@ -511,6 +514,76 @@ SQ8ComputeCodesL2Sqr(const uint8_t* codes1,
     return Generic::SQ8ComputeCodesIP(codes1, codes2, lowerBound, diff, dim);
```

Comment:
lowerBound -> lower_bound

Reply:
this issue is belongs to SQ8, it should be fixed in another PR?

Reply:
OK